benchmarks/src/btreemap.rs

```diff
@@ -219,6 +221,53 @@ pub fn btreemap_insert_10mib_values() -> BenchResult {
     })
 }
 
+#[bench]
+pub fn btreemap_iter_count() -> BenchResult {
+    let mut btree = BTreeMap::new(DefaultMemoryImpl::default());
+    let size: u8 = 200;
+    for i in 0..size {
+        btree.insert(i, i);
+    }
+
+    benchmark(|| {
+        for i in 0..size {
+            for j in i + 1..size {
+                btree
+                    .range((Bound::Included(i), Bound::Included(j)))
+                    .count();
+            }
+        }
```

Comment:
What do we gain by having this loop. Why not simplify to something like this?

```suggestion
        btree.range((Bound::Included(0), Bound::Included(size))).count();
```

Reply:
We are checking if the count works for all possible ranges in the BTreeMap. We are not getting a lot, but we are a bit more sure it works properly. And since there are not that many ranges I was thinking that it is good to have it this way.